OrbitGl/App.cpp

```diff
@@ -961,6 +963,19 @@ void OrbitApp::OnProcessSelected(uint32_t pid) {
   }
 }
 
+//-----------------------------------------------------------------------------
+bool OrbitApp::GetUploadDumpsToServerEnabled() const {
+  return GParams.m_UploadDumpsToServer;
+}
+
+//-----------------------------------------------------------------------------
+void OrbitApp::EnableUploadDumpsToServer(bool a_Value) {
+  options_.crash_handler->SetUploadsEnabled(a_Value);
```

Comment:
Can you check whether options_.crash_handler is not a nullptr before calling SetUploadsEnabled?